index.html

```diff
@@ -1965,7 +1880,7 @@ <h2>Holder Acts on Behalf of the Subject</h2>
       <pre class="example nohighlight" title="An example of the relationship
 property in a child's credential">{
   "id": "http://dmv.example.gov/credentials/3732",
-  "type": ["VerifiableCredential", "ProofOfAgeCredential"],
+  "type": ["VerifiableCredential", "ChildProofOfAgeCredential"],
```

Comment:
Did you mean to change this?

Reply:
Yes, I made this comment many weeks ago in an issue and it seemed to get overlooked.
Rationale: the contents of this VC are different from the other VCs that are proof of age, therefore the type should be different